core/sds-test-aspect-models/src/main/resources/valid/bamm_2_0_0/io.openmanufacturing.test.shared/1.0.0/AspectWithCollectionEntity.ttl

```diff
@@ -0,0 +1,52 @@
+# Copyright (c) 2022 Robert Bosch Manufacturing Solutions GmbH
+#
+# See the AUTHORS file(s) distributed with this work for additional
+# information regarding authorship.
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#
+# SPDX-License-Identifier: MPL-2.0
+
+@prefix : <urn:bamm:io.openmanufacturing.test.shared:1.0.0#> .
+@prefix bamm: <urn:bamm:io.openmanufacturing:meta-model:2.0.0#> .
+@prefix bamm-c: <urn:bamm:io.openmanufacturing:characteristic:2.0.0#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix unit: <urn:bamm:io.openmanufacturing:unit:2.0.0#> .
+
+:AspectWithEntity a bamm:Aspect ;
+   bamm:preferredName "Test Aspect"@en ;
+   bamm:description "This is a test description"@en ;
+   bamm:see <http://example.com/omp> ;
+   bamm:properties ( :testProperty ) ;
+   bamm:operations ( ) .
+
+:testProperty a bamm:Property ;
+   bamm:preferredName "Test Property"@en ;
+   bamm:description "This is a test property."@en ;
+   bamm:see <http://example.com/omp> ;
+   bamm:see <http://example.com/me> ;
+   bamm:characteristic :EntityCharacteristic .
+
+:EntityCharacteristic a bamm-c:SingleEntity ;
+   bamm:preferredName "Test Entity Characteristic"@en ;
+   bamm:description "This is a test Entity Characteristic"@en ;
+   bamm:see <http://example.com/omp> ;
+   bamm:dataType :TestEntity .
+
+:TestEntity a bamm:Entity ;
+   bamm:extends :ParentTestEntity ;
+   bamm:preferredName "Test Entity"@en ;
+   bamm:description "This is a test entity"@en ;
+   bamm:properties ( :entityProperty ) .
+
+:entityProperty a bamm:Property ;
+   bamm:preferredName "Entity Property"@en ;
+   bamm:description "This is a property for the test entity."@en ;
+   bamm:characteristic bamm-c:Text .
+
+:ParentTestEntity
```

Comment:
Adjust formatting here to the rest of the file please

Reply:
done